catalog/solutions/nri-cts/0.9.15/solution.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "nri-cts",
+  "name": "NRI-CTS",
+  "version": "0.9.15",
+  "website": "https://nri-cts.net",
+  "provider": "Nomura Research Institute, Ltd.",
```

Comment:
The value of the `"provider"` field should rather be a `<user-id>`, with lower case letters and dashes, rather than whitespace. The name of the company will then be fetched from the user `json` file (which we are also requesting). 
Could you please replace this value with one in the valid format (e.g., `"nri"` or `"nomura-research-institute"`)?

Reply:
Isn't the `name` in `solution.json` supposed to be the name of our product? If so, NRI-CTS is fine.
I think the one corresponding to the search keywords is `summary` in `solution.json`, is it not?